app/src/main/res/values/strings-preferences.xml

```diff
@@ -29,6 +30,10 @@
     <string name="pref_key__http_proxy_port" translatable="false">pref_key_proxy_port</string>
     <string name="pref_key__proxy_was_enabled" translatable="false">wasProxyEnabled</string>
 
+    <!-- Operability -->
+    <string name="pref_title__topbar_stream_shortcut">Top toolbar loads stream</string>
```

Comment:
I would maybe leave it in `Visuals` currently. We have to resort those settings anyway at some point